src/validation.cpp

```diff
@@ -4886,8 +4887,8 @@ bool DumpMempool(const CTxMemPool& pool)
 }
 
 //! Guess how far we are in the verification process at the given block index
```

Comment:
Should move this comment to the header or remove it?

Reply:
Removed.